docs/config.json

```diff
@@ -681,6 +681,10 @@
               "title": "Session Locking",
               "slug": "/access-controls/guides/locking/"
```

Comment:
If we're removing the u2f guide, we should make sure there's a redirect here.

Reply:
Well spotted. Done.